addon/helpers/list.js

```diff
@@ -2,6 +2,16 @@ import { MicroState } from 'ember-microstates';
 
 export default MicroState.extend({
 
+  initialValueFor([array]) {
+    if (array === undefined) {
+      return [];
+    } else if (array && array.length != null && array.forEach) {
```

Comment:
Believe it or not, we already have a scenario: That's when you want to keep appending files to a list because of a drag and drop or a file input. The data comes in as a `FileList` which has a `length` and `[index]` working, but it does not implement `forEach`